article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -176,14 +174,16 @@ object ArticlePicker {
       LocalRenderArticle
     } else if (dcrShouldRender(request)) {
       RemoteRender
-    } else if (dcrCouldRender(page, request) && userIsInCohort) {
+    } else if (dcrCouldRender(page, request) && userIsInDotcomRenderingCohort && ArticlePageChecks.isDiscussionDisabled(page)) {
+      RemoteRender
+    } else if (dcrCouldRender(page, request) && userIsInDiscussionRenderingCohort) {
```

Comment:
I'm clinging on here so may well be way off but won't `dcrCouldRender` always fail if discussion articles aren't whitelisted yet? Meaning neither of these conditions will fire for comment articles?

Would it work if we updated `dcrShouldRender` to something like:

```scala
  def dcrShouldRender(request: RequestHeader): Boolean = {
    // dcrShouldRender provides an override to let us force rendering by DCR even
    // when an article is not supportted
    val userIsInDiscussionRenderingCohort = ActiveExperiments.isParticipating(DiscussionRendering)
    val forceDCR = request.forceDCR

    forceDCR || isInWhitelist(request.path) || userIsInDiscussionRenderingCohort
  }
```
?

Reply:
You might need to finesse that to only trigger if the article has comments but `dcrShouldRender` feels like the right place for this logic to happen

Reply:
Um.... Let me merge it as it is because it sort of make sense for the moment, and let me know if you see a case where the logic will need to be updated :)

Reply:
@oliverlloyd I think what you missed was `("isDiscussionDisabled", ArticlePageChecks.isDiscussionDisabled(page)),` being removed from the whitelist and added directly to the if statement above?